lib/puppetlabs_spec_helper/module_spec_helper.rb

```diff
@@ -22,12 +22,10 @@ def verify_contents(subject, title, expected_lines)
   begin
     require 'simplecov'
     require 'simplecov-console'
-    require 'codecov'
 
     SimpleCov.formatters = [
       SimpleCov::Formatter::HTMLFormatter,
       SimpleCov::Formatter::Console,
-      SimpleCov::Formatter::Codecov,
     ]
     SimpleCov.start do
       track_files 'lib/**/*.rb'
```

Comment:
Wait, how did this ever get coverage for puppet files?

Reply:
Not. This is only for ruby code.